README.rst

```diff
@@ -32,21 +32,49 @@ Javascript. With Batavia, you can run Python bytecode in your browser.
 It honors Python 3.4.4+ syntax and conventions, and let's you
 reference objects and classes defined natively in JavaScript.
 
-Tutorial
---------
+Quick Start
+---------------
 
-To take Batavia for a spin, run through the `Getting Started guide`_.
+(For a more detailed setup, please see the `Getting Started tutorial <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_)
 
-Then have some fun with `the first tutorial`_, and try out running Python in
-your browser.
+Batavia requires a Python 3.4 installation, a copy of `Ouroboros <https://github.com/pybee/ouroboros>`_, and a virtualenv to run it all in. 
 
-.. _Getting Started guide: https://batavia.readthedocs.io/en/latest/intro/getting-started.html
-.. _the first tutorial: https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html
+Clone the code repos:: 
+
+   $ mkdir pybee
+   $ cd pybee
+   $ git clone https://github.com/pybee/batavia
+   $ git clone https://github.com/pybee/ouroboros
+
+Setup a virtualenv (for other environments, see `Getting Started <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_)::
+
+   $ virtualenv venv
+   $ . venv/bin/activate
+   $ cd batavia
+   $ pip install -e .
+
+And finally, run the :code:`make` script to generate the combined Batavia JavaScript files::
+
+   $ make
+
+Python in the Browser
+----------------------
+
+To run Batavia in the browser::
+
+    $ cd testserver
+    $ pip install -r requirements.txt
+    $ ./manage.py runserver
+
+then open a web browser at localhost:8080.
 
 Documentation
```

Comment:
- Make "Documentation for Batavia" act as the hyperlink to the root directoy of the RTD library (instead of "Ready the Docs")
- There seem to be a lot more sections in the RTD library, maybe consider listing the highest-level toc entries (like tutorials, how-to guide, etc).